tests/e2e/utils/cognito_bootstrap/cognito_pre_deployment.py

```diff
@@ -99,7 +99,7 @@ def create_cognito_userpool(
 
 # Step 3: Configure Ingress
 #TO DO: The current script fills in Helm values and Kustomize params.env files at the same time. Need to decouple the two in future.  
-def configure_ingress(cognito_userpool: CustomDomainCognitoUserPool, tls_cert_arn: str):
+def configure_ingress(cognito_userpool: CustomDomainCognitoUserPool, tls_cert_arn: str, load_balancer_scheme: str):
```

Comment:
set default to internet-facing here so that test doesnt need to change
https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/fixtures/cognito_dependencies.py#L91